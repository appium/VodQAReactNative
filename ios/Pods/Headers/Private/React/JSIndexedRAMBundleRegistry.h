../../../../../node_modules/react-native/ReactCommon/cxxreact/JSIndexedRAMBundleRegistry.h