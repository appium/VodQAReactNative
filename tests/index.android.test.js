import 'react-native';
import React from 'react';
import VodQAReactNative from '../index.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('VodQAReactNative iOS', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<VodQAReactNative />);
    expect(tree).toBeTruthy();
  });

  it('matches snapshot', () => {
    const tree = renderer.create(<VodQAReactNative />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
