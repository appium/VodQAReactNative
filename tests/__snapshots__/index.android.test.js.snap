// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VodQAReactNative iOS matches snapshot 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <View
    initialRoute={
      {
        "title": "Login",
      }
    }
    initialRouteStack={
      [
        {
          "title": "Login",
        },
        {
          "title": "Samples List",
        },
        {
          "title": "Carousel - Swipe left/right",
        },
        {
          "title": "Photos - Ping & Zoom",
        },
        {
          "title": "Webview",
        },
        {
          "title": "Slider",
        },
        {
          "title": "Vertical swiping",
        },
      ]
    }
    navigationBar={
      <NavigationBar
        routeMapper={
          {
            "LeftButton": [Function],
            "RightButton": [Function],
            "Title": [Function],
          }
        }
        style={
          {
            "backgroundColor": "#26aa5c",
          }
        }
      />
    }
    renderScene={[Function]}
  />
</View>
`;
